<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDataTransmissionCenter</class>
 <widget class="QWidget" name="AddDataTransmissionCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>数传中心参数</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>中心名称：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="centerName">
          <property name="maxLength">
           <number>40</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>FEP明文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="plaintextDestPortFep"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>中心IP地址：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="centerIP"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>FEP密文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="ciphertextDestPortFep"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>中心信宿：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="centerXS"/>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>FEP明文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="plaintextSrcPortFep"/>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>FEP密文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QSpinBox" name="ciphertextSrcPortFep"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>PDXP明文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="plaintextDestPortPDXP"/>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>FEP/RECP明文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QSpinBox" name="plaintextDestPortFepRECP"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>PDXP密文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="ciphertextDestPortPDXP"/>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>FEP/RECP密文目的端口：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QSpinBox" name="ciphertextDestPortFepRECP"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>PDXP明文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="plaintextSrcPortPDXP"/>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>FEP/RECP明文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QSpinBox" name="plaintextSrcPortFepRECP"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>PDXP密文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QSpinBox" name="ciphertextSrcPortPDXP"/>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>FEP/RECP密文源端口：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QSpinBox" name="ciphertextSrcPortFepRECP"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>描述：</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="3">
         <widget class="QPlainTextEdit" name="desc"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="confirmBtn">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
